apps/sms/views/conversation/js/conversation.js

```diff
@@ -2403,6 +2415,10 @@ var ConversationView = {
   setMessageStatus: function(id, status) {
     var messageDOM = document.getElementById('message-' + id);
 
+    if (!messageDOM) {
```

Comment:
Maybe `var messageDOM = document.getElementById('message-' + id) || this.buildingMessages[id];` ?